### PR TITLE
fix linting in cli/source/stub/stub_test.go

### PR DESCRIPTION
**Pull request by ukitta555**
### Description

Fix linting in stub_test.go
### Changelog
`no-changelog-required`

### Affected components

- [ ] Server
- [ ] Console
- [x] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

### Related Issues

#5831

### Solution and Design

Fixed linting errors. In general, just added error handling. ( if err != nil { t.Fatal(err) })

### Steps to test and verify

Run tests and check if everything is alright 👍 

### Limitations, known bugs & workarounds

N/A

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

**Comment by scriptonist** [comment]
@ukitta555 I think we don't need to declare a new error variable each time, maybe you declare a `err` variable and reuse it ?

File: cli/migrate/source/stub/stub_test.go [hunk 1]
```diff
@@ -17,16 +17,46 @@ func Test(t *testing.T) {
...
-	m.Append(&source.Migration{Version: 3, Direction: source.Up})
-	m.Append(&source.Migration{Version: 4, Direction: source.MetaUp})
-	m.Append(&source.Migration{Version: 5, Direction: source.Down})
-	m.Append(&source.Migration{Version: 6, Direction: source.MetaDown})
-	m.Append(&source.Migration{Version: 8, Direction: source.Up})
-	m.Append(&source.Migration{Version: 8, Direction: source.Down})
+	errUp := m.Append(&source.Migration{Version: 1, Direction: source.Up})
+	if errUp != nil {
+		t.Fatal(errUp)
+	}
```
Comment (scriptonist) [inline cli/migrate/source/stub/stub_test.go:23]:
```suggestion
	err := m.Append(&source.Migration{Version: 1, Direction: source.Up})
	if err != nil {
		t.Fatal(errUp)
	}
```

File: cli/migrate/source/stub/stub_test.go [hunk 2]
```diff
@@ -17,16 +17,46 @@ func Test(t *testing.T) {
...
+	errDown := m.Append(&source.Migration{Version: 1, Direction: source.Down})
+	if errDown != nil {
+		t.Fatal(errDown)
+	}
```
Comment (scriptonist) [inline cli/migrate/source/stub/stub_test.go:27]:
```suggestion
	err := m.Append(&source.Migration{Version: 1, Direction: source.Down})
	if err != nil {
		t.Fatal(errDown)
	}
```

**Review by scriptonist — CHANGES_REQUESTED** [review]

File: cli/migrate/source/stub/stub_test.go (hunk 2 above)
Comment (ukitta555) [reply cli/migrate/source/stub/stub_test.go]: fixed it! 👍

**Review by scriptonist — APPROVED** [review]

**Review by arvi3411301 — APPROVED** [review]

**Comment by mehtavishwa30** [comment]
Hello @ukitta555!

Thanks so much for your contribution to Hasura OSS during the Hacktoberfest 2020! We'd love to send you some Hasura swag as a token of appreciation. If you can share you email address, we will send you an email with the swag form shortly. ✨

**Comment by ukitta555** [comment]
> Hello @ukitta555!
> 
> Thanks so much for your contribution to Hasura OSS during the Hacktoberfest 2020! We'd love to send you some Hasura swag as a token of appreciation. If you can share you email address, we will send you an email with the swag form shortly. ✨

Hello!
It is ukitta555@gmail.com

**Comment by ukitta555** [comment]
Hello @mehtavishwa30 !
I didn't get the link to the form. Was I late with the email?

**Comment by mehtavishwa30** [comment]
Hey @ukitta555! Absolutely not!
Sorry, I was on a short break and just got back to work today. You should find your swag form in your mail shortly! 😉

**Comment by ukitta555** [comment]
> Hey @ukitta555! Absolutely not!
> Sorry, I was on a short break and just got back to work today. You should find your swag form in your mail shortly! 😉

Got it! Thanks a lot! 💯

**Comment by mehtavishwa30** [comment]
Awesome! Thanks for your patience! :)

**Pull request merged**